<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Template</class>
 <widget class="QWidget" name="Template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="persoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Espace Perso</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cardButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>La Carte</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectionButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SÃ©lection du chef</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>1201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="serveurButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="text">
       <string>Appeler Serveur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="commandButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Votre Commande</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>1211</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="centralLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
